searchform.php

```diff
@@ -1,8 +1,8 @@
-<form id="search" class="" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>"> 
+<form id="search" role="search" class="" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>"> 
   <div class="input-append">
 		<span aria-hidden="true" data-icon="&#xf002;"></span>
-		<input class="input-xlarge" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>">
+		<input label="Search" class="input-xlarge" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>"/>
```

Comment:
This is not the correct use of label; label is not an attribute on the `<input>` tag. Either use `aria-label`, or add an element to serve as the label and use standard `<label for`> syntax

Reply:
Fixed in a commit